src/util/error-handler.ts

```diff
@@ -151,3 +151,132 @@ export const reactionRolesErrorHandler: ErrorHandler = async (ctx: ButtonInterac
     
     // ----- !ERROR LOG -----
 };
+
+/**
+ * Actions related to the members logs category.
+ */
+export enum MemberLogsAction {
+    MemberJoin = "Member Join",
+    MemberLeave = "Member Leave",
+    MemberBan = "Member Ban",
+    MemberUnban = "Member Unban"
+}
+
+/**
+ * Actions related to the profiles logs category.
+ */
+export enum ProfileLogsAction {
+    MemberUpdate = "Member Update",
+    UserUpdate = "User Update"
+}
+
+/**
+ * Actions related to the messages logs category.
+ */
+export enum MessageLogsAction {
+    MessageEdit = "Message Edit",
+    MessageDelete = "Message Delete"
+}
```

Comment:
These are the enums for each moderation log action. They are split into the 3 moderation logs categories.